app/_sections/Features.tsx

```diff
@@ -5,37 +5,41 @@ export default function Features() {
     const features = [
         {
             title: "Multi-Layer Protection",
-            description: "Safeguarded by two distinct layers of security",
+            description: "Safeguarded by two distinct layers of security.",
             icon: <i className="uil uil-layer-group"/>,
         },
         {
-            title: "Ergo-Centric",
-            description: "Uses Ergo as a secure and cost efficient hub, reducing the need for several smart contracts on every chain. Facilitates linear growth complexity.",
+            title: "Smart Contract Reduction",
+            description: "Uses Ergo as a secure and cost efficient hub, reducing the need for several smart contracts on every bridged chain. Facilitates linear growth complexity.",
```

Comment:
It is already long. Can you summarize it a little?

Reply:
Try: Leverages Ergo for security and cost savings, minimizing smart contracts on bridged chains and ensuring scalable growth.